cms/templates/course_info.html

```diff
@@ -45,7 +45,7 @@ <h1 class="page-header">
         <h3 class="sr">${_('Page Actions')}</h3>
         <ul>
           <li class="nav-item">
-            <a href="#" class=" button new-button new-update-button"><span class="icon fa fa-plus" aria-hidden="true"></span> ${_('New Update')}</a>
+            <a href="#" class=" button new-button new-update-button" disabled><span class="icon fa fa-plus" aria-hidden="true"></span> ${_('New Update')}</a>
```

Comment:
To clarify-- waiting until a-links are converted to buttons throughout Studio (which is a pervasive issue and will require CSS changes too).